model/resource/service_experimental.yaml

```diff
@@ -1,6 +1,7 @@
 groups:
   - id: service_experimental
     type: resource
+    name: 'service'
```

Comment:
it seems that resource name is not unique (e.g. we have the same name for experimental and stable). But we use it such for back-compat checks.

Would it create any problems when merging resources?

Reply:
backcompat is only checking stable resources against each other.

Right now, you can/should only move stable attribues from experimental -> stable group, so it should be ok.

I agree that there's some oddities here, but I think we can make progress on better resource/entity modelling in parallel.  For now, let's protect the bits we stabilized.